toonz/sources/CMakeLists.txt

```diff
@@ -249,9 +280,9 @@ if(PLATFORM EQUAL 32)
     add_subdirectory(t32bitsrv)
 endif()
 
-if (APPLE)
+if(APPLE OR UNIX)
```

Comment:
Makes no sense, on OS X both `APPLE` and `UNIX` would be set. https://cmake.org/cmake/help/v3.0/variable/UNIX.html

`if (UNIX)` will suffice.